examples/apps/colorapp/servicemesh/config/virtualservices/colorgateway-virtualservice.json

```diff
@@ -0,0 +1,10 @@
+{
+    "spec": {
+        "provider": {
+            "virtualRouter": {
+                "virtualRouterName": "colorgateway-vr"
+            }
+        }
+    },
+    "virtualServiceName": "colorgateway@@SERVICES_DOMAIN@@"
```

Comment:
Why not just colorgateway? do the service need to have SERVICES_DOMAIN in their name?

Reply:
Actually, color gateway doesn't really need a VirtualService or a VirtualRouter as it's accessed directly.

The only reason for keeping @@SERVICES_DOMAIN@@ in general is it's also mapped to the actual DNS namespace, no? So without it on `colorteller`, requests would not be initiated.

Reply:
I think it makes sense to keep the VRouter and VService for gateway for now, just in case we decide to build something on top.